biojava-structure/src/test/java/org/biojava/nbio/structure/io/mmcif/TestChemCompProvider.java

```diff
@@ -7,14 +7,19 @@
 import java.io.InputStream;
 import java.nio.file.Path;
 import java.nio.file.Paths;
+import java.util.logging.Logger;
 
 import org.biojava.nbio.structure.Structure;
+import org.biojava.nbio.structure.StructureException;
+import org.biojava.nbio.structure.StructureIO;
+import org.biojava.nbio.structure.align.util.AtomCache;
 import org.biojava.nbio.structure.align.util.UserConfiguration;
 import org.biojava.nbio.structure.io.FileParsingParameters;
 import org.biojava.nbio.structure.io.PDBFileReader;
 import org.junit.Test;
 
 public class TestChemCompProvider {
+	private static final Logger s_logger = Logger.getLogger( TestChemCompProvider.class.getPackage().getName() );
```

Comment:
Same here, use slf4j logger as we do elsewhere in biojava. Otherwise the logging config won't work well for this.